osu.Framework/Audio/Track/Track.cs

```diff
@@ -23,6 +23,8 @@ public abstract class Track : AdjustableAudioComponent, ITrack, IAudioChannel
 
         public virtual bool Looping { get; set; }
 
+        public Tags? Tags { get => GetTags(); }
```

Comment:
What's the point of this property-method split? Is it for encapsulation? I'm not super sure it's worth it as opposed to

```csharp
        public virtual Tags? Tags => null;
```

If really required though:

```suggestion
        public Tags? Tags => GetTags();
```